src/unified_ai_api.py

```
@@ -1843,7 +1837,8 @@
             return
 
     except Exception as e:
-        await websocket.close(code=4001, reason=f"Authentication failed: {str(e)}")
+        logger.error(f"WebSocket authentication failed: {str(e)}", exc_info=True)
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)